sentinel-demo/sentinel-demo-dubbo/pom.xml

```diff
@@ -12,16 +12,21 @@
     <artifactId>sentinel-demo-dubbo</artifactId>
 
     <dependencies>
-        <dependency>
-            <groupId>org.springframework</groupId>
-            <artifactId>spring-context</artifactId>
-            <version>5.0.7.RELEASE</version>
-        </dependency>
         <dependency>
             <groupId>com.alibaba</groupId>
             <artifactId>dubbo</artifactId>
             <version>2.6.5</version>
         </dependency>
+        <dependency>
```

Comment:
Maybe we can add a comment here to explain why the dependency is needed here.

Reply:
Yes, it's needed. Comments has added, please check.
PS: dubbo 2.6.6 has not released in maven repository so far.